src/arweave_address.test.ts

```diff
@@ -38,5 +38,22 @@ describe('The ArweaveAddress class', () => {
 			const address = new ArweaveAddress('g1hzNXVbh2M6LMQSUYp7HgkgxdadYqYEfw-HAajlms0');
 			expect(`${address}`).to.equal('g1hzNXVbh2M6LMQSUYp7HgkgxdadYqYEfw-HAajlms0');
 		});
+
+		it('throws if casted to a number', () => {
+			const address = new ArweaveAddress('g1hzNXVbh2M6LMQSUYp7HgkgxdadYqYEfw-HAajlms0');
+			expect(() => +address).to.throw();
+		});
+
+		it('address comparison: same address', () => {
```

Comment:
Test names should read like a fluent English sentence. Your test above `throws if casted to a number` works great in its final context: `ArweaveAddress class interpolated toString function throws if casted to a number.`

But the example on this line does not read fluently:
`ArweaveAddress class interpolated toString function address comparison: same address`
When I read that I don't understand what the behavior is supposed to be. So as a reviewer or a code reader, I start from a confused perspective.

Lastly, your tests are really for the `equalsAddress` function, so they should be in a describe block specifically for that. Altogether, the test might read as:
`ArweaveAddress class equalsAddress function returns true for matching addresses`

Reply:
Got ya! c46b2c41e0727e676b0d35671fd38b705b2cd30b